burotel/indico_burotel/plugin.py

```diff
@@ -35,19 +50,68 @@ def _add_missing_time(field, data, time_):
         data[field] = dts.dump({field: datetime.combine(date, time_)})[field]
 
 
+def _check_update_permissions(booking):
+    if booking.room.attributes.filter(
+        RoomAttribute.name == 'electronic-lock',
+        RoomAttributeAssociation.value == cast('yes', JSONB)
+    ).join(Room.attributes, RoomAttributeAssociation.attribute).first():
+        @after_this_request
+        def _launch_task(response):
+            update_access_permissions.delay(booking)
+            return response
+
+
+def _check_no_parallel_bookings(booking):
+    """Ensure that the user has no other bookings in that interval."""
+    overlapping = query_user_overlapping_bookings(booking).all()
+    if overlapping:
+        raise ExpectedError(_("There is a parallel booking for this person in {0}, from {1} to {2}").format(
+            overlapping[0].room.full_name,
+            format_date(overlapping[0].start_dt),
+            format_date(overlapping[0].end_dt)
+        ))
+
+
+class SettingsForm(IndicoForm):
+    _fieldsets = [
+        ('ADaMS Sync', ['adams_service_url', 'adams_username', 'adams_password', 'cern_identity_provider'])
```

Comment:
```suggestion
        (_('ADaMS Sync'), ['adams_service_url', 'adams_username', 'adams_password', 'cern_identity_provider'])
```

(even though we don't even actively translate those plugins..)